tests/api-diff/run.t

```diff
@@ -35,24 +35,27 @@ Here we generate a basic `.mli` files with two types and a function:
 
 ### A file with an additional type:
 
-  $ cat > add_type.mli << EOF
+  $ cat > add_type.mli <<EOF
   > type t = int
   > type added_t = float
   > val f : t -> string
-  > EOF 
-
+  > EOF
+ 
 We generate the .cmi file
 
   $ ocamlc add_type.mli
 
 Now we run api-watcher on the two cmi files, there should be a difference:
 
   $ api-diff ref.cmi add_type.cmi
-  API changed!
+  api-watcher: internal error, uncaught exception:
+               Includemod.Error(_)
+               
+  [125]
 
 ### A file with a removed type:
 
-  $ cat > remove_type.mli << EOF
+  $ cat > remove_type.mli <<EOF
```

Comment:
Here you are removing everything, not just a type! Can you make it so the only diff between this and the reference is a removed type?

Reply:
Okay thank you for the review. Working on that asap!